node/cmd/guardiand/adminnodes.go

```diff
@@ -116,6 +116,7 @@ func runListNodes(cmd *cobra.Command, args []string) {
 		{"Sei", vaa.ChainIDSei},
 		{"Scroll", vaa.ChainIDScroll},
 		{"Mantle", vaa.ChainIDMantle},
+		{"Movement", vaa.ChainIDMovement},
```

Comment:
I think this list is missing some of the previously added chains. Would be good to bring it in line (and maybe add some consistency to the sorting).

Reply:
Don't want to cloud this PR with that stuff. Will do a follow-on PR.